frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py

```diff
@@ -294,7 +294,7 @@ def mix(is_equal, if_equal, case_1, case_2, case_3):
     #
     # There is a single output of equal, it goes to input 0 of mix
     composition = fhe.Wired(
-        [
+        {
```

Comment:
maybe, if you need to change uses of modules, have a look to SHA1, who also uses modules, please

```
grep -r "fhe\.module" frontends/concrete-python/examples 
```

```
frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py:    module: fhe.module  # type: ignore
frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py:@fhe.module()
frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py:def levenshtein_simulate(module: fhe.module, x: str, y: str):
frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py:def levenshtein_fhe(module: fhe.module, x: str, y: str):
frontends/concrete-python/examples/sha1/sha1.py:@fhe.module()
frontends/concrete-python/examples/sha1/README.md:@fhe.module()
```

Reply:
Ah my bad ! Will do !